src/SampleWebApp/Startup.cs

```diff
@@ -41,6 +42,11 @@ public Startup(IHostingEnvironment env, IApplicationEnvironment appEnv)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            services.AddDbContext<AuthorDataContext>(options =>
+                    options.UseSqlServer(@"Server=(localdb)\mssqllocaldb;Database=AuthorDataContext-9a1d8034-a6d6-4cdb-bf59-77e85aad38ac;Trusted_Connection=True;MultipleActiveResultSets=true"));
+
+            services.AddDbContext<BookDataContext>(options =>
+                    options.UseSqlServer(@"Server=(localdb)\mssqllocaldb;Database=BookDataContext-586e0d4b-10df-4ce2-b3a7-8d98621d6dcf;Trusted_Connection=True;MultipleActiveResultSets=true"));
```

Comment:
This is a known issue from RC1. We do plan to move the connection strings to appsettings.json for RTW.